prow/plugins/trigger/ic_test.go

```diff
@@ -107,15 +131,79 @@ func TestHandleIssueComment(t *testing.T) {
 		// Trusted member's test this.
 		{
 			Author:      "t",
-			Body:        "@k8s-bot test this",
+			Body:        "/test all",
```

Comment:
i didn't remove these test cases for `/test all` here because the duplicated test cases have been removed from `jobs_test.go`.